### PR TITLE
RA-1998

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js [hunk 1]
```diff
@@ -738,22 +735,9 @@ export default class ProgramStatusController {
         this.$window.location.reload();
     }
 
-    dateWithoutTime(date) {
-        return moment(date).format('YYYY-MM-DD');
-    }
-
-    toUTCDate(dateString) {
```
Comment (mogoodrich) [inline omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js:745]: Note that this toUTCDate function was added to fix a similar issue to this issue (see RA-1774) however, as far as I can tell, the assumption this function was making was that the server dates are always in UTC, which may not be the case)

**Comment by mogoodrich** [comment]
Updated PR ready for review, for description see: https://issues.openmrs.org/browse/RA-1998

**Review by mseaton — APPROVED** [review]
All this looks well done to me @mogoodrich .  It would be good to get some input from those who added the original handling of this feature, to provide some additional review and (ideally) testing, once merged.

**Comment by mogoodrich** [comment]
Very good point @mseaton ... I had mean to flag @mks-d  but forgot and added him now... @mks-d  I believe (but I could be wrong) that the initial fix was applied for the Mekom use case.  Is there anyone at Mekom that can review and/or test once it's merged in.

**Review by mks-d — COMMENTED** [review]
@mogoodrich please go ahead. I don't think this widget is still currently used on our end. Thanks for the heads up.

**Comment by mogoodrich** [comment]
Thanks all, merging!

**Pull request merged**